examples/features/xds/README.md

```diff
@@ -0,0 +1,32 @@
+# gRPC xDS example
```

Comment:
Should there be something at the top explaining what xds even is?

Reply:
I think we will need a more comprehensive document/user-guide about xDS and how to use it. We can add that later.

Reply:
OK, let's try to remember to circle-back at that time and add a link from here.